server/cmd/museum/main.go

```diff
@@ -712,7 +712,7 @@ func main() {
 }
 
 func runServer(environment string, server *gin.Engine) {
-	if environment == "local" {
+	if environment == "local" || environment == "proxy" {
```

Comment:
@ua741 This is the only source change (apart from packaging a proxy.yaml, duplicate of production.yaml). This felt like a more robust solution to start with instead of using config params, but can tinker too. Untested yet, will do a test deployment after building from this branch.